beacon_node/client/src/builder.rs

```diff
@@ -590,7 +591,7 @@ where
                 })?
         };
 
-        self.eth1_service = None;
+        self.eth1_service = Some(backend.core.clone());
```

Comment:
Any idea why we set this to `None` earlier? curious again :)

Reply:
Hmm.. no I can't :sweat_smile: